demo/app/wasm.go

```diff
@@ -11,6 +11,8 @@ func AllCapabilities() []string {
 		"cosmwasm_1_1",
 		"cosmwasm_1_2",
 		"cosmwasm_1_3",
+		"cosmwasm_1_4",
+		"cosmwasm_2_0",
```

Comment:
It seems `cowsmwasm_2_0` implies all of the preceding ones. See the https://github.com/CosmWasm/cosmwasm/blob/main/MIGRATING.md#15x---20x bullets.

Reply:
Hmm it seems that we still need all of these -- it's the CW contract that the `cosmwasm_2_0` feature implies the prior versions. If the [chain](https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES-BUILT-IN.md) wants to support CW contract with lower CW versions we still need those capabilities. Keeping them for now